tests/test_robot_states_flatcache.py

```diff
@@ -63,8 +58,8 @@ def camera_pose_test(flatcache):
         relative_pose_transform(sensor_world_pos, sensor_world_ori, robot_world_pos, robot_world_ori)
     )
 
-    sensor_world_pos_gt = np.array([150.16513062, 150.0, 101.38952637])
-    sensor_world_ori_gt = np.array([-0.29444987, 0.29444981, 0.64288363, -0.64288352])
+    sensor_world_pos_gt = np.array([150.16513062, 150.0, 101.39360809])
```

Comment:
presumably updated due to numerical discrepancies?

Reply:
any way to make this less brittle?

Reply:
TODO: maybe record gt values and then call og.clear() to check for alignment